app.js

```diff
@@ -0,0 +1,16 @@
+var express = require("express"),
```

Comment:
I think it should end with ";" not ','

Reply:
var express = require("express"),
      app = express();
or
var express = require("express");
var	app = express();

2 different syntax in JS.
I changed it according to what you told me.